pkg/util/i18n/translations/en-us.go

```diff
@@ -0,0 +1,16 @@
+package translations
+
+var EnUSFileName = "en-US.all.json"
+
+var EnUSData = []byte(`
```

Comment:
I inlined this JSON because I was hoping to avoid yet more gobindata, but I guess we'll have to go there...

I'm not opposed to conversion, is there a generally accepted format?  Making one up just so that we don't use JSON doesn't seem worth it...